site/docs/components/progress/examples.mdx

```diff
@@ -26,45 +26,36 @@ Use linear or circular depending on the context, layout and functionality of an
 
 </LivePreview>
 
-<LivePreview componentName="progress" exampleName="HiddenLabel" displayName="Hide label">
+<LivePreview componentName="progress" exampleName="LinearIndeterminate" displayName="Indeterminate linear progress">
 
-## Hide label
+## Indeterminate linear progress
 
-Use the `hideLabel` prop to display the progress without a label.
+Use an indeterminate linear progress to display an unspecified wait time such as a loading state.
 
 </LivePreview>
 
-<LivePreview componentName="progress" exampleName="WithMaxVal" displayName="With maximum value">
+<LivePreview componentName="progress" exampleName="WithMinVal" displayName="Custom min and max">
 
-## With maximum value
+## With minimum value
 
-Specify the maximum value of the progress indicator. The percentage progress will be calculated using this value. The default value is 100.
+Specify the `min` and `max` values of the progress indicator. The default range is 0 - 100.
 
 </LivePreview>
+<LivePreview componentName="progress" exampleName="WithBuffer" displayName="With value and buffer">
 
-<LivePreview componentName="progress" exampleName="WithMinVal" displayName="With minimum value">
-
-## With minimum value
+## With buffer
 
-Specify the minimum value of the progress indicator. The percentage progress will be calculated using this value. The default value is 0.
+Specify a buffer value to indicate loading state. The buffer does not have a label and will not affect the progress label.
```

Comment:
I'm not sure what "the buffer does not have a label" means?

Reply:
Updated slightly